.drone.yml

```diff
@@ -0,0 +1,527 @@
+---
+kind: pipeline
+name: default
+type: kubernetes
+
+environment:
+  APP_NAME: ima
+  PROD_ENV: ima
+  STG_ENV: ima-stg
+  UAT_ENV: sas-ima-uat
+  BRANCH_ENV: sas-ima-notprod
```

Comment:
ima-branch?

Reply:
Looking at [IMB-8](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-8), it is now sas-ima-branch so I have updated it and have updated ima-stg to sas-ima-stg